src/injected/visualization/svg-drawer.ts

```diff
@@ -265,11 +274,44 @@ export class SVGDrawer extends BaseDrawer {
         );
     }
 
-    private shouldBreakGraph(curElementIndex: number): boolean {
+    private createFocusIndicatorForFailure(item: TabbedItem): FocusIndicator {
+        const centerPosition: Point = this.centerPositionCalculator.getElementCenterPosition(
+            item.element,
+        );
+
+        if (centerPosition == null) {
+            return;
+        }
+
+        const drawerConfig: SVGDrawerConfiguration = this.formatter.getDrawerConfiguration(
+            item.element,
+            null,
+        ) as SVGDrawerConfiguration;
+
+        const circleConfiguration =
+            item.itemType === TabbedItemType.ErroredItem
+                ? drawerConfig.erroredCircle
+                : drawerConfig.missingCircle;
+
+        const newCircle = this.svgShapeFactory.createCircle(centerPosition, circleConfiguration);
+        const newLabel = this.svgShapeFactory.createTabIndexLabel(
+            centerPosition,
+            drawerConfig.tabIndexLabel,
+            item.tabOrder,
+        );
+
+        const focusIndicator: FocusIndicator = {
+            circle: newCircle,
+            tabIndexLabel: newLabel,
```

Comment:
Again, doesn't have to be in this PR but we will want a failure label here too: https://github.com/microsoft/accessibility-insights-web/pull/5199/files#diff-da79de6a20c6af768f69e58a29187f44ccf98e439534ee29d54dffcf8265f27cR306-R316